keymanapp/keyman
# chore(common): Check in crowdin strings for Mandara (Wandala)

This checks in the crowdin strings for the locale `mfi-NG` for Mandara (Wandala).

TODO:
~@sgschantz to check on adding the mfi-NG locale with XCode~
iOS locale not available

## User Testing
For all the platforms, load the PR build, set the UI to "Mandara (Wandala)"

* **TEST_ANDROID** - Tests Mandara UI for Android
1. Load the PR build in an Android 5.0 emulator (SDK 21)
2. In the Keyman app, go to Settings --> Display Language -->Mandara (Wandala)
3. Verify the UI is in Mandara

* **TEST_WINDOWS** - Tests Mandara UI for Windows

* **TEST_LINUX** - Tests Mandara UI for Linux
1.  start Keyman configuration with: `LANGUAGE=mfi_NG km-config`
